plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -423,6 +423,25 @@ class GesturesPluginTest {
     verify { cameraAnimationsPlugin.easeTo(any(), any()) }
   }
 
+  @Test
+  fun verifyHandleMoveExceptionFreeForInvalidFocalPoint() {
+    mockkStatic("com.mapbox.maps.MapboxLogger")
+    every { logE(any(), any()) } just Runs
+    val moveGestureDetector = mockk<MoveGestureDetector>()
+    every { moveGestureDetector.pointersCount } returns 1
+    every {
+      moveGestureDetector.focalPoint
+    } returns PointF(Float.NaN, Float.NaN)
+    var handled = presenter.handleMove(moveGestureDetector, 50.0f, 50.0f)
```

Comment:
nit: seems to be as several parameterized tests.
and should you verify `logE(...)` was called?

Reply:
imo a bit too much for param tests here but added verify for calling logger, thanks!